### PR TITLE
add the ssticky header style and put it as first thing after body ele…

### PR DESCRIPTION
**Pull request by VladDubrovskis**
…ment
/cc @adgad @gvonkoss @andrewgeorgiou1981

### CODE REVIEW

**Comment by VladDubrovskis** [comment]
Related n-ui PR:
https://github.com/Financial-Times/n-ui/pull/566

File: views/partials/ads/top.html [hunk 1]
```diff
@@ -1,7 +1,17 @@
-{{adslot "top"
-	cls="above-header-advert o-ads--reserve-90 o-ads--background o-ads--transition"
-	pos="top"
-	medium="Leaderboard,Responsive"
-	large="SuperLeaderboard,Leaderboard,Responsive"
-	extra="Billboard,SuperLeaderboard,Leaderboard,Responsive"
-}}
+{{#if @root.flags.stickyHeaderAd}}
```
Comment (adgad) [inline views/partials/ads/top.html]: could just do the if statement within the `cls`?

Comment (VladDubrovskis) [inline views/partials/ads/top.html]: did not find a way for this. maybe but I dont know :)

**Comment by adgad** [comment]
👍

**Pull request merged**